src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -0,0 +1,1882 @@
+							activateSomeComponents();
+						} else {
+							MessageDialog.error(this,
+								"angal.inventory.cannotcreateanotherinventorywithotherinprogressinthisward.msg");
+						}
+					}
+				}
+			});
+		}
+		return wardComboBox;
+	}
+
+	private void disabledSomeComponents() {
+		jCalendarInventory.setEnabled(false);
+		specificRadio.setEnabled(false);
+		codeTextField.setEnabled(false);
+		allRadio.setEnabled(false);
+		referenceTextField.setEnabled(false);
+		jTableInventoryRow.setEnabled(false);
+		saveButton.setEnabled(false);
+		deleteButton.setEnabled(false);
+		resetButton.setEnabled(false);
+		lotButton.setEnabled(false);
+	}
+
+	private void activateSomeComponents() {
+		jCalendarInventory.setEnabled(true);
+		specificRadio.setEnabled(true);
+		codeTextField.setEnabled(true);
+		allRadio.setEnabled(true);
+		referenceTextField.setEnabled(true);
+		jTableInventoryRow.setEnabled(true);
+		wardComboBox.setEnabled(false);
+		saveButton.setEnabled(true);
+		deleteButton.setEnabled(true);
+		resetButton.setEnabled(true);
+		lotButton.setEnabled(true);
+	}
+
+	private JLabel getLoaderLabel() {
+		if (loaderLabel == null) {
+			Icon icon = new ImageIcon("rsc/icons/oh_loader.GIF");
+			loaderLabel = new JLabel("");
+			loaderLabel.setIcon(icon);
+			loaderLabel.setVisible(false);
+		}
+		return loaderLabel;
+	}
+
+	private void addMedInRowInInventorySearchList(MedicalInventoryRow inventoryRow) {
+		int position = getPosition(inventoryRow);
+		if (position == -1) {
+			position = inventoryRowSearchList.size();
+			inventoryRowSearchList.add(position, inventoryRow);
+		} else {
+			inventoryRowSearchList.add(position + 1, inventoryRow);
+		}
+		if (inventoryRow.getId() == 0) {
+			inventoryRowListAdded.add(inventoryRow);
+		}
+	}
+
+	private int getPosition(MedicalInventoryRow inventoryRow) {
+		int position = -1;
+		int i = 0;
+		for (MedicalInventoryRow inventoryRow1 : inventoryRowSearchList) {
+			if (inventoryRow1.getMedical().getCode().equals(inventoryRow.getMedical().getCode())) {
+				position = i;
+			}
+			i++;
+		}
+		return position;
+	}
+
+	private void resetVariable() {
+		inventoryRowsToDelete.clear();
+		lotsDeleted.clear();
+		inventoryRowListAdded.clear();
+		lotsSaved.clear();
+	}
+
+	private boolean isLotWithCost() {
+		return GeneralData.LOTWITHCOST;
+	}
+
+	private boolean isAutomaticLotIn() {
+		return GeneralData.AUTOMATICLOT_IN;
+	}
+
+	protected double askTotalCost() {
+		String input = JOptionPane.showInputDialog(this, MessageBundle.getMessage("angal.medicalstock.multiplecharging.totalcost"), 0.);
+		double total = 0.;
+		if (input != null) {
+			try {
+				total = Double.parseDouble(input);
+				if (total < 0) {
+					throw new NumberFormatException();
+				}
+			} catch (NumberFormatException nfe) {
+				MessageDialog.error(this, "angal.medicalstock.multiplecharging.pleaseinsertavalidvalue");
+			}
+		}
+		return total;
+	}
+
+	private Lot getLot(Lot lotToUpdate) throws OHServiceException {
+		Lot lot = null;
```

Comment:
```suggestion
		Lot lot;
```
Assignment to `null` is redundant.